src/mz/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "mz"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { version = "1.0.137", features = ["derive"] }
+clap = { version = "3.2.6", features = [ "derive" ] }
+tokio = { version = "1.19.2", features = ["full"] }
+open = "3.0.1"
+actix-web = "4.1.0"
```

Comment:
Our HTTP server of choice is `axum`. Mind swapping this one out so that we don't have two separate HTTP servers floating around the codebase?

Reply:
Done! I should have looked at Materialize code before choosing Actix. At first, I was in doubt about which one to pick and ended up with the most "used".